packages/api/cms-api/src/translation/azure-ai-translator.resolver.ts

```diff
@@ -0,0 +1,45 @@
+import createClient, { isUnexpected, TextTranslationClient } from "@azure-rest/ai-translation-text";
+import { Inject } from "@nestjs/common";
+import { Args, Query, Resolver } from "@nestjs/graphql";
+
+import { RequiredPermission } from "../user-permissions/decorators/required-permission.decorator";
+import { AzureAITranslatorConfig } from "./azure-ai-translator.config";
+import { AZURE_AI_TRANSLATOR_CONFIG } from "./azure-ai-translator.constants";
+import { TranslationInput } from "./dto/translation.input";
+
+@Resolver()
+@RequiredPermission(["translation"], { skipScopeCheck: true })
+export class AzureAITranslatorResolver {
+    private translationClient: TextTranslationClient;
+    constructor(@Inject(AZURE_AI_TRANSLATOR_CONFIG) private readonly config: AzureAITranslatorConfig) {
```

Comment:
```suggestion

    constructor(@Inject(AZURE_AI_TRANSLATOR_CONFIG) private readonly config: AzureAITranslatorConfig) {
```

Nit: Empty line before constructor